### DISCUSSION TITLE
Need help for installation accelerate config

### DISCUSSION THREAD

**Discussion by HysTakuya** [bmaltais/kohya_ss#106] [Q&A]
Hi,
I encountered a problem during the installation, when I wish to choose mixed precision (FP16 or BF16) I press down arrow button then i saw this messages.
I've tried twice.

OS: win11
Python 3.10

thanks in advance

![mixed precision issue](https://user-images.githubusercontent.com/124450072/216774831-f69457b4-d0bd-4c75-9421-7db8deb4c086.png)

**Comment by davidyich** [comment]
@HysTakuya faced the same problem. The solution at this step is to press the number 1, and then enter

**Reply by HysTakuya** [reply]
> @HysTakuya faced the same problem. The solution at this step is to press the number 1, and then enter

Yes, it seems I can only choose no to compelte installation, but I can change it in the WEBUI.
